internal/model/model.go

```diff
@@ -135,7 +135,7 @@ func (m Model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 }
 
 func (m Model) View() string {
-	r, _ := glamour.NewTermRenderer(m.Theme, glamour.WithWordWrap(0))
+	r, _ := glamour.NewTermRenderer(m.Theme, glamour.WithWordWrap(m.viewport.Width))
```

Comment:
I think this approach is better, what do you think?

Reply:
I tried this but I had a weird behaviour. I'll post some screenshots to show you the difference.

Reply:
Actually, now that you mention it, I think I turned off this wrapping probably for the same weird behaviours you're talking about.

If you have screenshots that would be awesome. We can probably make an issue upstream in glamour in that case.

Reply:
Sorry for the delay. I cannot find the examples anymore... When there are too much lines it does not work well (the header does not show), but otherwise it does not mess the alignment as it did before.

Maybe the glamour version bump helped as well ?

Reply:
Yea possibly they may have fixed it upstream, I'll test this out a bit and get this merged